website/docs/deployment.mdx

```diff
@@ -397,3 +397,25 @@ echo subdomain.surge.sh > CNAME
 ```
 
 You can deploy any other changes in the future with the command `surge`.
+
+## Deploying to QuantCDN
+
+## QuantCDN
```

Comment:
```suggestion
```

deleting duplicate heading

Reply:
@steveworley I can't commit to your PR so will wait for you to apply this suggestion